### PR TITLE
Create CNAME

### PR DESCRIPTION
**Pull request by aljaxus**

### CODE REVIEW

**Review by AlexProgrammerDE — DISMISSED** [review]
lgtm!

**Comment by aljaxus** [comment]
Mkay - so this part of CI renders this MR to an utterly useless state; https://github.com/SkinsRestorer/SkinsRestorerX/blob/main/.github/workflows/deploy.yml#L27-L31

**Comment by AlexProgrammerDE** [comment]
Actually this folder is being used instead because of github pages l allowing to select the /docs folder.

**Review by xknat — APPROVED** [review]
👍

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -26,9 +26,11 @@ jobs:
         run: mvn -B site --file pom.xml
       - name: Move files
         run: |
-          rm -rf docs/
```
Comment (xknat) [inline .github/workflows/deploy.yml:29]: Are you sure that we remove this?

Comment (aljaxus) [reply .github/workflows/deploy.yml:29]: Naw it's good because it'll be run from other branches, not `gh-pages` which has the docs contents

**Pull request merged**